cfme/tests/configure/test_access_control.py

```diff
@@ -162,6 +163,16 @@ def test_user_remove_tag():
     user.delete()
 
 
+def test_delete_default_user():
+    user = ac.User(name='Administrator')
+    sel.force_navigate("cfg_accesscontrol_users")
+    row = records_table.find_row_by_cells({'Full Name': user.name})
```

Comment:
It looks like the row has slightly different header? Looks to me that in 5.3, there is no `Full Name` header, only `Name`.